hooks/command

```diff
@@ -135,6 +135,19 @@ if plugin_read_list_into_result BUILDKITE_PLUGIN_DOCKER_VOLUMES BUILDKITE_PLUGIN
   done
 fi
 
+# Ensure devices option is an array
```

Comment:
I don't think we need these array checks anymore, because `plugin_read_list_into_result` already checks it and outputs an error.

Reply:
Thanks for the swift feedback. I am happy to remove this instance of it but perhaps a more through cleanup is a better approach?

Reply:
Sure thing! Let's just go with it.

Reply:
Cool, can you please confirm that accepting `boolean` for `volumes` is a mistake? so that I can safely remove that check too.